sysdig/internal/client/v2/cloudauth/go/cloud_account.pb.go

```diff
@@ -140,6 +142,64 @@ func (Feature) EnumDescriptor() ([]byte, []int) {
 	return file_cloud_account_proto_rawDescGZIP(), []int{1}
 }
 
+type ValidationResult int32
```

Comment:
@cgeers can you ack that these validation changes are intended too?
just clarifications for later review, since is not related to WIF needs